asab/library/service.py

```diff
@@ -318,12 +318,14 @@ async def _read_disabled(self):
 		for k, v in disabled.items():
 			if k.endswith('/'):
 				self.DisabledPaths.append((k, v))
+				self.Disabled[k] = v
```

Comment:
Why?
Can you comment on the reason, please?

Reply:
The line self.Disabled[k] = v was added to ensure that directory-based disablements are not only tracked in the self.DisabledPaths but also reflected in the self.Disabled dictionary. This is crucial because of synchronization with .disabled.yaml File ,  since we use self.Disabled to store the disabled information in the .disabled.yaml file, including directory disabled's in self.Disabled ensures that these are accurately represented in the .disabled.yaml.

Reply:
I was testing the feature with Franta, I made this a temporary fix so we could test it. I will undo those changes if you disagree.

Reply:
It has to be handled properly in asab-library.